### DISCUSSION TITLE
Installation bug

### DISCUSSION THREAD

**Discussion by JackqqWang** [open-mmlab/mmclassification#1325] [Q&A]
I am following the link to install
https://mmclassification.readthedocs.io/en/dev-1.x/get_started.html#install-from-source
However, I met an error as below:
![WeChat Screenshot_20230128211847](https://user-images.githubusercontent.com/64749583/215300822-000a2438-1498-4798-8346-576a2b262933.png)
My environment is:
Python 3.9.7
NAME="Ubuntu"
VERSION="20.04.5 LTS (Focal Fossa)"
May I ask what could fix it?
Thanks

**Comment by Ezra-Yu** [comment]
you can go on installation and verify the installation refer to [doc](https://mmclassification.readthedocs.io/en/dev-1.x/get_started.html#verify-the-installation). Is there any error?
